### DISCUSSION TITLE
How to not use TOR to connect to watchdog services?

### DISCUSSION THREAD

**Discussion by fiatjaf** [ACINQ/eclair#1947] [Node operators]
My `eclair.conf` has this:

![2021-09-10-095555_233x191_scrot](https://user-images.githubusercontent.com/1653275/132856628-5fd6e4bd-c058-40d1-9472-ce7778dd52a7.png)

But occasionally I get errors on my console that suggest I am using TOR to connect to websites:

![2021-09-10-095716_1178x522_scrot](https://user-images.githubusercontent.com/1653275/132856756-d199dd2d-8642-4dd1-a9af-5942949731e2.png)

I want to use normal HTTP, what should I do?

**Comment by t-bast** [comment]
Good point, this is a regression introduced by https://github.com/ACINQ/eclair/pull/1907
@rorp can you fix this? You actually need to check `use-for-tor` before choosing to use tor for watchdogs.
